index.html

```diff
@@ -38,18 +38,25 @@ <h2 data-i18n="app.title"></h2>
             <div class='col12' id='address-step'>
                 <div class='center pad2'>
                     <span id='instructions' data-i18n="step1.desc"></span>
-                    <p id="couldnt-find" style='display: none;' data-i18n="[html]step1.fail"></p>
                 </div>
                 <div class='col12'>
                     <div class='pad1'>
                         <form id='find'>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
+                            <br />
+                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
```

Comment:
Since the geolocation API won't work on non-ssl connections, can we think of a way to hide this if they're on http? I'll make sure onosm redirects to SSL, but just in case...